cypress/integration/userCanSearchDrinks.feature.js

```diff
@@ -0,0 +1,38 @@
+describe("User can search", () => {
+  beforeEach(() => {
+    cy.server();
+    cy.visit("http://localhost:3001");
+  });
+  it("search field exists", () => {
+    cy.get("#name-search.prompt").should("exist");
+    cy.get("#search").should("exist");
+    cy.get("#search").contains("Search");
+  });
+  it("successfully by drink name", () => {
+    cy.route({
+      method: "POST",
+      // url: "http://localhost:3000/api/v1/cocktails",
```

Comment:
I would work with `baseUrl` so that you don't have to change the host. Also, remove commented-out code before final review.

Reply:
We tried playing with baseUrl but maybe do not fully understand how to utilize its full potential. We will try to get it to work but may need to reach out for assistance.

Reply:
The baseUrl is now used instead of the full http call out.